wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/elevatorprofiledpid/Robot.java

```diff
@@ -25,6 +26,8 @@ public class Robot extends TimedRobot {
       new TrapezoidProfile.Constraints(1.75, 0.75);
   private final ProfiledPIDController m_controller =
       new ProfiledPIDController(1.3, 0.0, 0.7, m_constraints, kDt);
+  private final ElevatorFeedforward m_feedforward = 
+      new ElevatorFeedforward(1, 1.1, 1.2);
```

Comment:
Extract these (and other constants) to a constants file, or at least constants at the top of the class.